community/kernel/src/main/java/org/neo4j/kernel/NeoStoreDataSource.java

```diff
@@ -1123,6 +1142,14 @@ public synchronized void stop()
         // fail since the logFile no longer works.
     }
 
+    private void awaitAllTransactionsClosed()
+    {
+        while ( !neoStoreModule.neoStore().closedTransactionIdIsOnParWithOpenedTransactionId() )
+        {
+            LockSupport.parkNanos( 1_000_000 ); // 1 ms
```

Comment:
Waiting less than 10 milliseconds will change the system-wide process scheduling quantum on Windows.